apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -120,10 +120,19 @@ export const UnconnectedJsDebugger = Radium(React.createClass({
       watchersHidden: false,
       open: this.props.isOpen,
       openedHeight: 120,
+      labelDisplay: 'inline'
     };
   },
 
+  handleWindowResize() {
+    let hideLabel = ($(window).width() <= 1275) && !this.state.watchersHidden;
+    let labelDisplay= hideLabel ? 'none' : 'inline';
```

Comment:
nit: missing space before =

Also nit, when possible I tend to lean towards having state be more conceptual rather than directly tied to the view. i.e. Instead of having the state be a CSS setting, have it be something like `labelHidden` and then in your render have `style={[this.state.labelHidden && styles.hidden, styles.noUserSelect]}`